src/DataAccessLayer/dalMain.cs

```diff
@@ -40,10 +40,10 @@ public CosmosDal(Secrets secrets, Config config)
                 CosmosUrl = secrets.CosmosServer,
                 Retries = config.Retries,
                 Timeout = config.Timeout,
-            };
 
-            // create the CosmosDB client and container
-            cosmosDetails.Client = OpenAndTestCosmosClient(secrets.CosmosServer, secrets.CosmosKey, secrets.CosmosDatabase, secrets.CosmosCollection).GetAwaiter().GetResult();
+                // create the CosmosDB client and container
+                Client = OpenAndTestCosmosClient(secrets.CosmosServer, secrets.CosmosKey, secrets.CosmosDatabase, secrets.CosmosCollection).GetAwaiter().GetResult(),
```

Comment:
Any reason why Client is set within the object but container is set outside? This is throwing an error in the container build

Reply:
Just to silence Message	IDE0017	Object initialization can be simplified. 
Interesting that error does get thrown when running it in the Visual Studio IDE
I will suppress the message in C# code with comments.